Source/Core/Core/AchievementManager.cpp

```diff
@@ -682,6 +685,24 @@ void AchievementManager::HandleAchievementTriggeredEvent(const rc_runtime_event_
                                 Config::Get(Config::RA_ENCORE_ENABLED));
 }
 
+void AchievementManager::HandleAchievementProgressUpdatedEvent(
+    const rc_runtime_event_t* runtime_event)
+{
+  if (!Config::Get(Config::RA_PROGRESS_ENABLED))
+    return;
+  auto it = m_unlock_map.find(runtime_event->id);
+  if (it == m_unlock_map.end())
+    return;
+  AchievementId game_data_index = it->second.game_data_index;
+  FormattedValue value{};
+  if (rc_runtime_format_achievement_measured(&m_runtime, runtime_event->id, value.data(),
```

Comment:
`if` statements that span multiple lines should wrap things (even if one line) in `{` and `}`